application.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'dataset' # Data of countries
+require_relative 'country' # Country Class
+
+def main
+
+  countries_array = [] # array to store each country object
+
+  $countries_data.each do |country, data|
+    country_obj = Country.new(data[0],data[1],data[2],data[3],data[4],data[5])
+    countries_array << country_obj
```

Comment:
Instead of countries_array, you could directly iterate over the collection returned by each.